README.md

```diff
@@ -21,7 +21,7 @@ Read in different language : [**zh**](localization/zh/README.md), [**ko**](local
 
 # Introduction
 
-Design patterns are the best formalized practices a programmer can use to
+Design patterns are the best-formalized practices a programmer can use to
```

Comment:
I think the intention here is to say `Design patterns are the best, formalized practices...`

Reply:
Ok correcting.